packages/astro/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "declarationDir": "./dist",
     "outDir": "./dist",
     "jsx": "preserve",
-    "types": ["@types/dom-view-transitions"]
+    "types": ["@types/dom-view-transitions"],
+    "rootDir": "src"
```

Comment:
The compiler complained about ambiguous rootDir for some reason

Reply:
I remember facing this as well, I think you need to revert the import change from `astro/zod` to `zod`

Reply:
Thanks! That did the trick. Well remembered